examples/sparsestream/server/server.py

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import asyncio
+
+import yaml
+
+import uvicorn
+from deepsparse.transformers import pipeline
+from fastapi import FastAPI
+from fastapi.routing import APIRouter
+from fastapi_websocket_pubsub import PubSubEndpoint
+from tweepy.asynchronous import AsyncStream
+from usernames import user_id
+
+
+token_path = "./server/config.yaml"
+model_path = "zoo:nlp/text_classification/bert-base/pytorch/huggingface/sst2/base-none"
```

Comment:
```suggestion
model_path = "zoo:nlp/sentiment_analysis/bert-base/pytorch/huggingface/sst2/12layer_pruned80_quant-none-vnni"
```

Reply:
FYI, the sentiment analysis will eventually be a multi-class (3 labels) model. This is just a placeholder.